test_data/asm/vm_args_relative_path.asm

```diff
@@ -1,4 +1,9 @@
-machine Main with degree: 16 {
+// we cannot have `N` at the top level because of how shorts paths are displayed with `.` instead of `::`
```

Comment:
shorts paths?

Reply:
This is an existing issue where `a::b` gets printed as `a.b` to support both PIL and asm. The workaround here is to have a longer path.

Reply:
Ah ok. "Short path" then

Reply:
done